index.html

```diff
@@ -68,151 +75,115 @@ <h2 class="lets-connect">let's connect</h2>
         </li>
         <li>
          
-            <img src="./images/Vector.png" alt="vector">
+            <img src="./images/Mail.png" alt="vector">
 
     </div>
 
-
+  </div>
   </section>
     <!-- works -->
 
     
 <section class="works">
   
-<div class="card-works">
-
-       <div class="all-image">
-          <img src="./images/Snapshoot1.1.png"   alt="">
-       </div>
-
-  <div class="dsk-2">
+<div class="card-works container">
+  <img id="mobile-1" class="first-image" src="./images/Snapshoot Portfolio.off.png" alt="">
 
-       <h2 class="headings">Tonics</h2>
+  <div class="card-details">
+    <h2 class="headings">Tonics</h2>
 
-          <ul class="first-item">
-               <li class="one"> <a href="#">Canopy</a> </li>
-                <li class="other"> <a href="#">back end dev</a></li>
-               <li class="other"> <a href="#">2015</a></li>
-         </ul>
-
-   <p class="text-1"> 
-
-    A daily selection of privately personalized reads; no accounts or sign-ups required.
+ <ul class="first-item">
+   <li> Canopy </li>
+   <li> back end dev</li>
+   <li> 2015</li>
+ </ul>
 
+  <p class="text-1">
+    A daily selection of privately <br>
+     personalized reads; no accounts or sign-ups required.
   </p>
 
-         <ul class="item-2">
-                  <li class="link-item"> <a href="#"></a> html</li>
-                  <li  class="link-item" > <a href="#">css</a></li>
-                  <li class="link-item"> <a href="#">javascript</a></li>
-          </ul>
+  <ul class="item-2">
+      <li> html</li>
+      <li>css</li>
+      <li>javascript</li>
+  </ul>
 
-         <p class="see">
-              <a href="#">See projects</a>
-         </p>
-
-  </div>
+   <button class="btn">see projects</button>
 
+    </div>
   </div>
 
-  <!-- first card ends-->
-
-   <div class="card-works-rev">
-    <div class="all-image">
-      <img src="./images/Snapshoot2.png" alt="">
-    </div>
-    
-      <div class="dsk-2"> 
-          <p class="headings">
-          Multi-Post
-          Stories
-      </p>
+<!--first  card ends-->
 
-      <ul class="first-item">
+<div class="card-works container">
+  <img  class="first-image" src="./images/now.png" alt="">
 
-            <li class="one"> <a href="#">Canopy</a> </li>
-            <li class="other"> <a href="#">back end dev</a></li>
-            <li class="other"> <a href="#">2015</a></li>
+ <div class="card-details"> 
+ <h3 class="headings"> Multi-Post <br> Stories</h3>
 
```

Comment:
- [ ] 🙏Kindly align your projects using the correct layout, taking into consideration whether the image is on the left or right side of the card. This will give your projects an impressive appearance🌹 reminiscent of the Figma design.

Your design:
![image](https://github.com/hetrox8/portfolio1/assets/63445083/e478a726-8e0c-4349-97b8-bc763901daf1)
![image](https://github.com/hetrox8/portfolio1/assets/63445083/ea50ab00-6571-47d6-9aa5-c4dec068a97c)

Figma design:
![image](https://github.com/hetrox8/portfolio1/assets/63445083/e28182f5-1100-4bb6-a5fe-34e374430ae3)
![image](https://github.com/hetrox8/portfolio1/assets/63445083/39ca361b-df58-4aa5-8fc7-37cab991ece7)